src/main/scala/com/databricks/labs/overwatch/env/Workspace.scala

```diff
@@ -53,6 +53,24 @@ class Workspace(config: Config) extends SparkSessionWrapper {
 
   }
 
+  // The below function is supposed to fetch data about DatabricksSQLEndPoints
+  def getDatabricksSqlDF: DataFrame = {
+
+    val jobsEndpoint = "sql/endpoints"
```

Comment:
This is ONE of the endpoints. The key here is to figure out which data is ONLY available via the API calls and which data can otherwise be derived from the auditLogs. We want to use the audit logs where possible and fallback on API calls only where we have to.

Reply:
One of the most important endpoints is going to be the `"sql/history/queries"` and we actually need to pass in a query as well since there's a hidden query to get the metrics. The query string for the get request is `include_metrics=true`. 

Note: The APICall logic already enables passing queries, you just need to pass in the query in the right format.

Reply:
There are also some important differences in the Global SQL Endpoints API. I am almost certain we'll need a couple of snapshot tables for endpoints and one or two appenders to capture all of the SQL endpoint data in bronze.